fiatconnect-api.md

```diff
@@ -2227,12 +2228,56 @@ A KYC schema containing personal data about a user, as well as documents such as
 	},
 	phoneNumber: `string`,
 	selfieDocument: `string`,
-	identificationDocument: `string`
+    identificationDocument: `string`
 }
 ```
 
 The `selfieDocument` and `identificationDocument` fields should be base64 encoded binary blobs representing images.
 
+#### 9.3.1.2. `PersonalDataAndDocumentsDetailed`
+A more detailed Schema allowing the user to provide more information to work with CICO providers that require it.
+
+```
+{
+	firstName: `string`,
+	middleName?: `string`,
+	lastName: `string`,
+	dateOfBirth: {
+		day: `string`,
+		month: `string`,
+		year: `string`
+	},
+	address: {
+		address1: `string`,
+		address2?: `string`,
+		isoCountryCode: `string`,
+		isoRegionCode: `string`,
+		city: `string`,
+		postalCode?: `string`
+	},
+	phoneNumber: `string`,
+    email: `string`,
+	selfieDocument: `string`,
+    identificationDocumentType: `enum`,
+	identificationDocumentFront: `string`,
+    identificationDocumentBack?: `string` 
+}
+```
+
+The `phoneNumber` field is REQUIRED and MUST follow the formating of the [E.164 international standard](https://en.wikipedia.org/wiki/E.164). It MUST contain all parts of the phone number INCLUDING the Country Code.
+
+The `email` field MUST be a [valid email](https://en.wikipedia.org/wiki/Email_address#Syntax).
+
+The accepted documents can be found [here](https://en.wikipedia.org/wiki/Identity_document). The document should fit one of the `identificationDocumentType` field options:
```

Comment:
Can we formally add an `IdentificationDocumentType` enum to the spec? It would be consistent with patterns used elsewhere in the spec, and we would presumably create an `IdentificationDocumentType` enum in the `fiatconnect-types` repo anyhow, so it would be good to have it mirrored in the specifcation.